pom.xml

```diff
@@ -879,7 +879,7 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-javadoc-plugin</artifactId>
-				<version>2.9.1</version>
+				<version>3.0.1</version>
```

Comment:
It would probably be better to have a property that holds the version than repeat it. That makes it easier to upgrade.

Reply:
I'm going to fix it. thank you .